src/astro/sql/operators/load_file.py

```diff
@@ -116,6 +122,11 @@ def load_data_to_dataframe(self, input_file: File) -> Optional[pd.DataFrame]:
                 df = pd.concat([df, file.export_to_dataframe()])
             else:
                 df = file.export_to_dataframe()
+
+            df = convert_dataframe_col_case(
+                df=df, columns_names_capitalization=self.columns_names_capitalization
+            )
```

Comment:
I believe we would benefit from keeping this logic inside the database class itself, within `export_to_dataframe` - since our motivation to mess up with capitalization is because of issues with a specific database

Reply:
@tatiana I think this functionality spans the database as we are doing this for the dataframe operator as well. This PR handles this functionally for all the places a dataframe is created.

Reply:
As per discussion with @tatiana we agreed that this functionality spans across the lib, wherever we are creating the dataframe and this param introduces a way to control cols case. Hence resolving this comment.